assets/src/blocks/amp-story-page/index.js

```diff
@@ -107,13 +108,17 @@ export const settings = {
 			overlayStyle.opacity = overlayOpacity / 100;
 		}
 
+		const imgStyle = {
+			objectPosition: IMAGE_BACKGROUND_TYPE === mediaType && focalPoint ? `${ focalPoint.x * 100 }% ${ focalPoint.y * 100 }%` : 'initial',
```

Comment:
This is mainly taken [from edit.js](https://github.com/ampproject/amp-wp/blob/0a883ce83c15c22407bebef69db1bd5808d07247/assets/src/blocks/amp-story-page/edit.js#L162).